workers/api-server/src/http.rs

```diff
@@ -122,6 +123,8 @@ const PEER_ID_URL_PARAM: &str = "peer_id";
 const TASK_ID_URL_PARAM: &str = "task_id";
 /// The :matching_pool param in a URL
 const MATCHING_POOL_URL_PARAM: &str = "matching_pool";
+/// The ?matching_pool param in a query string
+const MATCHING_POOL_QUERY_PARAM: &str = "matching_pool";
```

Comment:
nit: just use the constant above it? rename if you think the name isn't general enough